packages/@aws-cdk/aws-lambda/lib/lambda.ts

```diff
@@ -1,4 +1,5 @@
 import iam = require('@aws-cdk/aws-iam');
+import { Queue, QueueRef } from '@aws-cdk/aws-sqs';
```

Comment:
We prefer namespaced imports (I know we have some "oldish" code that used this style, but we are trying to move away from it):

```ts
import sqs = require('@aws-cdk/aws-sqs');
```

Reply:
done